src/components/nodes/utilities/distribute.js

```diff
@@ -0,0 +1,74 @@
+import { moveShapeLeftTo, moveShapeMiddleXTo, moveShapeMiddleYTo } from '@/components/nodes/utilities/shapeMovement';
+import { getBoundingBox } from '@/components/nodes/utilities/shapeGroup';
+
+export function distributeVerticalCentersEvenly(shapes) {
+  const bounds = getBoundingBox(shapes);
+  const itemCount = shapes.length;
+  if (itemCount < 3) {
```

Comment:
Should this `3` be a variable explaining why that is the limit?

Reply:
Yeah, thanks :)